app/pages/lab/data-dumps.cjsx

```diff
@@ -24,12 +24,23 @@ module.exports = React.createClass
   getInitialState: ->
     avatarError: null
     backgroundError: null
+    showAggregationExportButton: false
+
+  componentDidMount: ->
+    @showAggregationExport @props.project
 
   showWorkflowExport: ->
     Dialog.alert(
       <ExportWorkflowsDialog project={@props.project} />
     )
 
+  showAggregationExport: (project) ->
```

Comment:
This is duplicating the API request that is handled by `<DataExportButton />`. Instead, you'll need to conditionally determine if the API call should be made, hide it if it never has, disable the button for any new exports, but still render the link to download the previous export. Basically, the changes need to happen in the `<DataExportButton />` which is not ideal, but should only be temporary until we communicate to project builders that this feature is going to be disabled at least until we have a better aggregation solution. The help text in this component can also probably be removed.